tests/__init__.py

```diff
@@ -0,0 +1,37 @@
+"""Unit test module."""
+import os
+import sys
+import unittest
```

Comment:
Não sabia que o Python tinha um built-in de tests, interessante.

Reply:
Pois é, não conhecia também